### PR TITLE
Added booster to HAS action

### PR DESCRIPTION
**Pull request by hskim881028**
- Added booster to `HackAndSlash` action

Related Links 
- https://app.asana.com/0/958521740385861/list
- https://www.notion.so/planetarium/d6d58782022347d1b1a3a7fdff1d4af5#0650562bc96148e893923ef0e9ec7754

### CODE REVIEW

File: Lib9c/Action/HackAndSlash.cs [hunk 1]
```diff
@@ -156,15 +159,22 @@ public override IAccountStateDelta Execute(IActionContext context)
             Log.Verbose("{AddressesHex}HAS get CostumeStatSheet: {Elapsed}", addressesHex, sw.Elapsed);
             sw.Restart();
 
-            if (avatarState.actionPoint < stageRow.CostAP)
+            if (playCount <= 0)
```
Comment (ipdae) [inline Lib9c/Action/HackAndSlash.cs:162]: we need check stage already cleared when playCount > 1

**Comment by sonohoshi** [comment]
Can add `playCount` field in BattleLog class?
I want to use it in 9c - Stage.cs. like this.
```cs
var countableItem = new CountableItem(item, _battleLog.playCount); // In development branch, Stage.cs, 930 line
```
In addition to this, there are many situations to refer to `playCount`.

File: Lib9c/Action/HackAndSlash.cs (hunk 1 above)
Comment (hskim881028) [reply Lib9c/Action/HackAndSlash.cs:162]: Added. Please review again 🙏🏻 @ipdae

**Comment by ipdae** [comment]
test failed

**Review by ipdae — COMMENTED** [review]
i think it would be good to add 2 test case
- execution hackAndSlash use booster(count > 1) 
- unit test for new get reward

**Comment by hskim881028** [comment]
> i think it would be good to add 2 test case
> 
> * execution hackAndSlash use booster(count > 1)
> * unit test for new get reward

Added test code. considering the quest reward, the test code got longer. 😓 @ipdae

**Review by ipdae — APPROVED** [review]

**Pull request merged**